scripts/roles.json

```diff
@@ -3971,30 +4364,29 @@
     "prohibitedProps": [],
     "props": [
       "aria-atomic",
+      "aria-braillelabel",
+      "aria-brailleroledescription",
       "aria-busy",
       "aria-checked",
       "aria-controls",
       "aria-current",
       "aria-describedby",
+      "aria-description",
       "aria-details",
       "aria-disabled",
       "aria-dropeffect",
-      "aria-errormessage",
```

Comment:
This is still listed under this role as an allowed prop: https://www.w3.org/TR/wai-aria-1.2/#menuitemradio

Reply:
It's deprecated in 1.2: https://rawgit.com/w3c/aria/1d856c402eafbe7823021ff4cb5ca44dc1a9edf7/#menuitemradio

I did not include any deprecated attributes.

Reply:
@eps1lon we'll need to do a major-version release for these changes.